Firestore/Example/Tests/Integration/FSTDatastoreTests.mm

```diff
@@ -144,10 +144,13 @@ - (void)applySuccessfulWriteWithResult:(const MutationBatchResult &)batchResult
 }
 
 - (void)rejectFailedWriteWithBatchID:(BatchId)batchID error:(NSError *)error {
+  (void)batchID;
+  (void)error;
```

Comment:
Also, is there a term that refers to this patter of calling `(void)unusedParam`? I'm curious to know if this the commonly accepted way to handle unused vars that get passed down, or if there are other ways to do so.

Coming from TS patterns, this seems more excessive than just disabling the unused var linter for this line.